src/lib/support/JniReferences.h

```diff
@@ -160,6 +160,11 @@ class JniReferences
         return CreateBoxedObject(boxedTypeClsName, constructorSignature, chip::to_underlying(value), outObj);
     }
 
+    /**
+     * Fix to avoid NewStringUTF crash
```

Comment:
This comment does not help understand the code. This needs actual documentation.

Reply:
Comment edited.
By the way, according to your opinion below, it is more appropriate to handle Error. I will write again after the conversation below is resolved.